src/kb.py

```diff
@@ -16,17 +29,56 @@
 import wasabi
 from spacy import Vocab, Language
 from spacy.kb import KnowledgeBase, Candidate
-from spacy.tokens import Span
+from spacy.kb.candidate import BaseCandidate
+from spacy.tokens import Span, SpanGroup
 from spacy.util import SimpleFrozenList, SimpleFrozenDict
 
+from extraction.schemas import MentionEntity
 from extraction.utils import (
     establish_db_connection,
     load_entities,
 )
 
 
+class WikiKBCandidate(BaseCandidate):
```

Comment:
Yes. I plan to merge all of the functionality in https://github.com/explosion/wikid/pull/6 before updating `spacy@rmitsch:feature/candidate-generation-by-docs` (cf. https://github.com/explosion/spaCy/pull/11669) `spacy@explosion:develop`. This will be a separate PR with a number of changes to align `wikid` with the v4 spaCy EL codebase.